src/Network/Network/ExpressRoutePort/Authorization/RemoveAzureExpressRoutePortAuthorizationCommand.cs

```diff
@@ -0,0 +1,67 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Network.Models;
+using System.Linq;
+using System.Management.Automation;
+
+
+namespace Microsoft.Azure.Commands.Network
+{
+    [Cmdlet("Remove", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "ExpressRoutePortAuthorization", SupportsShouldProcess = true), OutputType(typeof(bool))]
+    public class RemoveAzureExpressRoutePortAuthorizationCommand : NetworkBaseCmdlet
+    {
+        [Parameter(
+            Mandatory = true,
+            HelpMessage = "The name of the Authorization")]
+        [ValidateNotNullOrEmpty]
+        public string Name { get; set; }
+
+        [Parameter(
+             Mandatory = true,
+             ValueFromPipeline = true,
+             HelpMessage = "The ExpressRoutePort")]
+        public PSExpressRoutePort ExpressRoutePort { get; set; }
```

Comment:
```suggestion
        public PSExpressRoutePort ExpressRoutePortObject { get; set; }
```
ExpressRoutePort  -> ExpressRoutePortObject to follow [piping-best-practices](https://github.com/Azure/azure-powershell/blob/main/documentation/development-docs/design-guidelines/piping-best-practices.md#long-explanation)